src/config.ts

```diff
@@ -424,9 +424,6 @@ export class ConfigurationReader implements vscode.Disposable {
         return this.configData.cpackArgs;
     }
     get configureOnOpen() {
-        if (util.isCodespaces() && this.configData.configureOnOpen === null) {
```

Comment:
You might want to keep a null check around for a while and flip it to true in case anyone accidentally put the default value in their settings.  You can remove the codespaces check though.

Reply:
Good idea. Done.